python/requirements.txt

```diff
@@ -7,3 +7,6 @@ pyjwt==1.6.4
 
 # Transformation of data, Apply JqFilter
 pyjq==2.2.0
+
+# openssl
+pyopenssl==19.0.0
```

Comment:
what is  the build size impact?

Reply:
@Singhal-Vaibhav can you please help in ascertaining pre and post FOGL-2471 build sizes?